sqlglot/expressions.py

```diff
@@ -2488,7 +2488,7 @@ class Least(Func):
 
 
 class Length(Func):
-    pass
+    arg_types = {"this": True}
```

Comment:
This is redundant -- (Func < Condition <) Expression already defines arg_types like this.

Reply:
removed it, thank you